docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Overview of Linking, Deep Links, Android App Links, and iOS Universal Links
+sidebar_title: Overview
+description: An overview of available resources to implement Linking and Deep Links in your Expo apps.
+---
+
+import { AndroidIcon } from '@expo/styleguide-icons/custom/AndroidIcon';
+import { AppleIcon } from '@expo/styleguide-icons/custom/AppleIcon';
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
+
+## Linking
+
+Linking allows your app to interact with incoming and outgoing URLs. In this process, the user not only gets directed to open your app, but they are taken to a specific screen (route) within the app.
+
+For example, by clicking an advertisement of your product, your app will open on the user's device and they can view that product's details. This product's link that the user clicked may look like:
+
+```html
+myapp://web-app.com/product
+```
+
+This link is constructed by three parts:
+
+- **Scheme**: The URL scheme that identifies the app that should open the URL (example: `myapp://`). It can also be `https` or `http` for non-standard deep links.
+- **Host**: The domain name of the app that should open the URL (example: `web-app.com`).
+- **Path**: The path to the screen that should be opened (example: `/product`). If the path isn't specified, the user is taken to the home screen of the app.
+
+## Linking strategies
+
+There are different linking strategies you handle in your Expo app:
+
+- Linking to other apps from your app (outgoing links)
```

Comment:
I might organize these differently to reinforce the differences between linking, deep linking, and the similarities between app links and universal links:
```
- Linking to other apps from your app (outgoing links)
- Linking to your app from other apps or a web browser (incoming links)
- Linking to specific content within your app (deep links)
- Redirecting links to your web domain to your app if it is installed (Android App Links and iOS Universal Links)
```